azkaban-common/src/main/c/execute-as-user.c

```diff
@@ -100,7 +105,7 @@ int main(int argc, char **argv){
 
     // try to change user
     fprintf(LOGFILE, "Changing user: user: %s, uid: %d, gid: %d\n", uid, user_info->pw_uid, user_info->pw_gid);
-    int retval = change_user(user_info->pw_uid, user_info->pw_gid);
+    int retval = change_user(uid, user_info->pw_uid, user_info->pw_gid);
     if (retval != 0){
         fprintf(LOGFILE, "Error changing user to %s\n", uid);
```

Comment:
Can we rename the local variable `uid` to `username` since it is not the `uid`.  Even the code
```
fprintf(ERRORFILE, "Requires at least 3 variables: ./execute-as-user uid command [args]");
```
is confusing since it mixes up `uid`, which is actually an `uint` vs `username` which is a string.